### PR TITLE
DOMA-7118 migrate ANALYTICS_V3 feature flag from organization to growthbook

### PR DESCRIPTION
**Pull request by sitozzz**

### CODE REVIEW

File: apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js [hunk 1]
```diff
@@ -69,9 +70,7 @@ describe('GetOverviewDashboardService', () => {
         organizationAdminUser = await makeClientWithNewRegisteredAndLoggedInUser()
         organizationAdminUser.organization = organization
 
-        await updateTestOrganization(admin, organization.id, {
-            features: [ANALYTICS_V3],
-        })
+        await updateTestOrganization(admin, organization.id)
```
Comment (SavelevMatthew) [inline apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js:73]: Why do you need an empty update?

File: apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js [hunk 2]
```diff
@@ -109,11 +108,13 @@ describe('GetOverviewDashboardService', () => {
         ticket = createdTicket
     })
 
+    beforeEach(() => {
```
Comment (SavelevMatthew) [inline apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js:111]: It's a bit strange to see `beforeEach` at the end of `describe` block :)

Comment (SavelevMatthew) [reply apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js:111]: Also should feature flag be toggled down in `afterEach`?

File: apps/condo/domains/organization/constants/features.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 const ALL_FEATURES = [
     SERVICE_PROVIDER_PROFILE_FEATURE,
-    ANALYTICS_V3,
 ]
 
 const ORGANIZATION_FEATURE_ENUM_NAME = 'OrganizationFeature'
 
 module.exports = {
     SERVICE_PROVIDER_PROFILE_FEATURE,
-    ANALYTICS_V3,
```
[the remark this answers is not in the document]
Comment (sitozzz) [reply apps/condo/domains/organization/constants/features.js:13]: Yes it should. I would write a migration script for that

File: apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js (hunk 2 above)
Comment (sitozzz) [reply apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js:111]: In most of the test cases feature flag should be enabled. But in a single test we should check if value of the `ANALYTICS_V3` flag is `false`, error was provided to the output.

From the other side, there is a case when developer want to run single test (for example with error about disabled feature). So the easiest way to make this work is enable feature before each test and disable feature if necessary

File: apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js (hunk 1 above)
Comment (sitozzz) [reply apps/condo/domains/analytics/schema/GetOverviewDashboardService.test.js]: That's my fault. Fixed it

**Review by SavelevMatthew — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**